.github/workflows/container-autoupdate.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Build and push container
         uses: docker/build-push-action@v2
         with:
-          context: dockerfile/ci-tasks/
-          file: dockerfile/ci-tasks/Dockerfile
+          context: dockerfile/anaconda-ci/
+          file: dockerfile/anaconda-ci/Dockerfile
```

Comment:
I suggest to rebase this PR to do the renaming first; then it's less confusing to look at.

Reply:
Fixed.